src/devices/Mcp23xxx/README.md

```diff
@@ -15,30 +15,83 @@ MCP23XXX devices contain different markings to distinguish features like interfa
 
 ## Binding Notes
 
+This binding includes an `Mcp23xxx` abstract class and implementations for both I2C (`Mcp230xx`) and SPI (`Mcp23Sxx`) interfaces.  You can create either one by passing in the respective communication driver.
+
+#### Mcp230xx I2C
+```csharp
+ // 0x20 is the device address in this example.
+var connectionSettings = new I2cConnectionSettings(1, 0x20);
+var i2cDevice = new UnixI2cDevice(connectionSettings);
+var mcp23Sxx = new Mcp230xx(i2cDevice);
```

Comment:
Should we change this readme to have a concrete class here instead of this abstract one? I suppose a lot of users will just copy paste code from this README and find that the code won't compile since you can't really new up an abstract class.

Reply:
Updated.  Thx